src/services/checkPassword.js

```diff
@@ -0,0 +1,11 @@
+const CheckPassword = function CheckPassword(input) {
+  const passw = /^(?=).{8,}$/;
+  if (passw.test(input) && input) {
```

Comment:
The second part of the condition has no purpose. If you want to check falsyness, then you should do that first.

Reply:
I still have, i put it as the first condition. Yes I am checking for falsyness